### DISCUSSION TITLE
Minimize to Systray

### DISCUSSION THREAD

**Discussion by neoOpus** [FlareSolverr/FlareSolverr#1171] [Ideas]
Could you please make it possible to minimize this application and have it start minimized in the system tray? I'd be happy to assist with creating an icon if needed.

**Comment by ilike2burnthing** [comment]
You can run it silently at startup, but currently no minimize to system tray functionality - https://github.com/FlareSolverr/FlareSolverr/issues/70#issuecomment-761954514
